plugin/pkg/admission/gc/gc_admission_test.go

```diff
@@ -30,6 +30,7 @@ import (
 	"k8s.io/apiserver/pkg/authorization/authorizer"
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	api "k8s.io/kubernetes/pkg/apis/core"
+	_ "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
```

Comment:
the tricky part is that, this test will be failing after removing internal clientset/informer from initializer. we have to manually init the legacy schema so that the [test rest mapper](https://github.com/kubernetes/kubernetes/pull/70167/files#diff-ab211ff9b4f7f8f6147c6719d598aadfR92) can work.

Reply:
> the tricky part is that, this test will be failing after removing internal clientset/informer from initializer. we have to manually init the legacy schema so that the [test rest mapper](https://github.com/kubernetes/kubernetes/pull/70167/files#diff-ab211ff9b4f7f8f6147c6719d598aadfR92) can work.

Blech.  That suggests that the test isn't quite testing what we want.  This worth an issue and a comment on the import line.